addons/stock/wizard/stock_picking_return.py

```diff
@@ -177,6 +177,9 @@ def _create_returns(self):
 
         new_picking.action_confirm()
         new_picking.action_assign()
+        for new_ml in new_picking.move_line_ids:
```

Comment:
we could do the following which is more efficient
```python 
for new_ml, old_ml in zip(new_picking.move_line_ids, move_id.origin_returned_move_id.move_line_ids):
    new_ml.package_id = old_ml.result_package_id
```
but i'm not sure the lines always match (specially in the case of partial returns)

Reply:
they don't